arednGettingStarted/node_status.rst

```diff
@@ -119,7 +119,7 @@ Status
   - *blocked - distance*: LQM determined that the remote node is either too close or too distant, based on the Min and Max Distance settings described in the *Advanced Configuration* section.
   - *blocked - signal*: LQM determined that the SNR on the link is too low to reliably pass data, based on the Min SNR setting described in the *Advanced Configuration* section.
   - *blocked - retries*: LQM determined that the retransmission rate was too high to reliably pass data.
-  - *blocked - latency*: LQM determined that the link latency was to long to reliably pass data.
+  - *blocked - latency*: LQM determined that the link latency was to high to reliably pass data.
```

Comment:
too high not to high

Reply:
Yes -- corrected on previous commit.  THX